samples/react-native/package.json

```diff
@@ -21,44 +21,45 @@
     "build-android-release-splits-flavors": "export SENTRY_SAMPLE_ENABLE_ABI_SPLIT=true; export SENTRY_SAMPLE_ENABLE_FLAVORS=true; cd android; ./gradlew assembleRelease; cd .."
   },
   "dependencies": {
-    "@react-navigation/bottom-tabs": "^7.0.4",
-    "@react-navigation/native": "^7.0.3",
-    "@react-navigation/native-stack": "^7.0.3",
-    "@react-navigation/stack": "^7.0.3",
+    "@react-navigation/bottom-tabs": "^7.2.0",
+    "@react-navigation/native": "^7.0.14",
+    "@react-navigation/native-stack": "^7.2.0",
+    "@react-navigation/stack": "^7.1.1",
     "@sentry/react-native": "6.6.0",
     "delay": "^6.0.0",
     "react": "18.3.1",
-    "react-native": "0.76.3",
-    "react-native-gesture-handler": "^2.21.1",
-    "react-native-reanimated": "3.16.1",
-    "react-native-safe-area-context": "4.14.0",
-    "react-native-screens": "4.1.0",
-    "react-native-svg": "^15.9.0",
+    "react-native": "0.77.0",
+    "react-native-gesture-handler": "^2.22.1",
+    "react-native-reanimated": "3.16.7",
+    "react-native-safe-area-context": "5.2.0",
+    "react-native-screens": "4.6.0",
+    "react-native-svg": "^15.11.1",
     "react-native-vector-icons": "^10.2.0",
-    "react-native-webview": "^13.12.3",
+    "react-native-webview": "^13.13.2",
     "react-redux": "^8.1.3",
     "redux": "^4.2.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.25.2",
-    "@babel/preset-env": "^7.25.3",
-    "@babel/runtime": "^7.25.0",
-    "@react-native-community/cli": "15.0.1",
-    "@react-native-community/cli-platform-android": "15.0.1",
-    "@react-native-community/cli-platform-ios": "15.0.1",
-    "@react-native/babel-preset": "0.76.3",
-    "@react-native/eslint-config": "0.76.3",
-    "@react-native/metro-config": "0.76.3",
-    "@react-native/typescript-config": "0.76.3",
+    "@babel/core": "^7.26.7",
+    "@babel/preset-env": "^7.26.7",
+    "@babel/runtime": "^7.26.7",
+    "@react-native-community/cli": "15.1.3",
+    "@react-native-community/cli-platform-android": "15.1.3",
+    "@react-native-community/cli-platform-ios": "15.1.3",
+    "@react-native/babel-preset": "0.77.0",
+    "@react-native/eslint-config": "0.77.0",
+    "@react-native/metro-config": "0.77.0",
+    "@react-native/typescript-config": "0.77.0",
     "@sentry/babel-plugin-component-annotate": "^2.18.0",
     "@types/react": "^18.2.65",
     "@types/react-native-vector-icons": "^6.4.18",
     "@types/react-test-renderer": "^18.0.0",
     "@typescript-eslint/eslint-plugin": "^7.18.0",
     "@typescript-eslint/parser": "^7.18.0",
-    "babel-jest": "^29.2.1",
+    "babel-jest": "^29.6.3",
     "babel-plugin-module-resolver": "^5.0.0",
     "eslint": "^8.19.0",
+    "eslint-plugin-ft-flow": "^3.0.11",
```

Comment:
Why do we need this devDependency?

Reply:
[There was a lint error](https://github.com/getsentry/sentry-react-native/actions/runs/13155598532/job/36712349827) pointing to this dependency after the `eslint-config` bump

```
  ESLint couldn't find the plugin "eslint-plugin-ft-flow".
  
  (The package "eslint-plugin-ft-flow" was not found when loaded as a Node module from the directory "/home/runner/work/sentry-react-native/sentry-react-native/samples/react-native".)
  
  It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:
  
      npm install eslint-plugin-ft-flow@latest --save-dev
  
  The plugin "eslint-plugin-ft-flow" was referenced from the config file in ".eslintrc.js » @react-native/eslint-config#overrides[0]".
```